model/export/SingleDeliveryResultsExporter.php

```diff
@@ -352,7 +352,7 @@ protected function getExporter(array $result)
      */
     protected function getExportData($exporter)
     {
-        return $exporter->export(true, false);
+        return $exporter->export(true, false,',','"',true);
```

Comment:
Where did the spaces between the args go? :)

Reply:
whoops :)